File: docs/code_snippet/java/restoreData.java
try {
   PurchasedItems purchasedItems = new PurchasedItems(/*context*/ this);
   String encryptedString = "--encrypted json string--";
   purchasedItems.restorePurchase(encryptedString);
   // using encrypted JSON object
   JSONObject encryptedJSON = // encrypted JSONObject
   purchasedItems.restorePurchase(encryptedJSON);
 } catch(JSONException e){
    e.printStackTrace();
}

File: docs/code_snippet/kotlin/restoreData.kt
try {
   val purchasedItems = PurchasedItems(/*context*/ this)
   val encryptedString = "--encrypted json string--";
   purchasedItems.restorePurchase(encryptedString);
   // using encrypted JSON object
   JSONObject encryptedJSON = // encrypted JSONObject
   purchasedItems.restorePurchase(encryptedJSON);
 } catch (e : JSONException){
    Log.e("Chapa", "Error: ${e.message}")
}